clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5410,7 +5406,9 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
   }
 
   for (const Arg *A :
-       Args.filtered(options::OPT_clang_ignored_legacy_options_Group)) {
+       // TODO: remove -fsycl-explicit-simd when support is dropped.
+       Args.filtered(options::OPT_clang_ignored_legacy_options_Group,
+                     options::OPT_fsycl_esimd, options::OPT_fno_sycl_esimd)) {
```

Comment:
The option doesn't look to be used anymore, so would it be better to modify the option in `Options.td` to be part of the `clang_ignored_legacy_options_Group` group?

Reply:
Right.